editor/questionbankcontroller.js

```diff
@@ -91,6 +89,7 @@ angular.module('createquestionapp', [])
     });
 
     $scope.searchQuestions = function (filterData, callback) {
+      $scope.itemsLoading = true;
```

Comment:
It should be "true" only when we are searching questions(only 1 place). It should hide after getting response(only 1 place). 

Remove from other unwanted places.